<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>lightSource</class>
 <widget class="QWidget" name="lightSource">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>240</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>160</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>打开串口</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>未打开</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>CH1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>CH2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>CH3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>CH4</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>200</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>光源通道</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>240</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>240</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>指定档位</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>221</width>
     <height>111</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>lightSource</receiver>
   <slot>openClose()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>154</x>
     <y>48</y>
    </hint>
    <hint type="destinationlabel">
     <x>153</x>
     <y>103</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBox_2</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>lightSource</receiver>
   <slot>channel_SW(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>66</x>
     <y>120</y>
    </hint>
    <hint type="destinationlabel">
     <x>180</x>
     <y>216</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>lightSource</receiver>
   <slot>levelSet()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>128</x>
     <y>179</y>
    </hint>
    <hint type="destinationlabel">
     <x>209</x>
     <y>174</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>openClose()</slot>
  <slot>channel_SW(int)</slot>
  <slot>levelSet()</slot>
 </slots>
</ui>
